main/background.ts

```diff
@@ -136,16 +136,28 @@ ipcMain.handle(
         const lines = fs.readFileSync(timereportPath, "utf8").split("\n");
         for (const line of lines) {
           const parts = line.split(" - ");
-          if (parts.length > 1 && parts[1] && parts[1] !== "!") {
-            latestProjects.add(parts[1]);
+          if (
+            !latesProjAndAct.hasOwnProperty(parts[1]) &&
+            parts[1] &&
+            !parts[1].startsWith("!")
+          ) {
+            latesProjAndAct[parts[1].trim()] = [""];
+          }
+          if (
+            parts.length > 3 &&
+            parts[1] &&
+            parts[1] !== "!" &&
+            !latesProjAndAct[parts[1]].includes(parts[2].trim())
+          ) {
+            latesProjAndAct[parts[1]].push(parts[2].trim());
```

Comment:
can we make 2 vars like const project = parts[1].trim() and const activity parts[2].trim()